groups.go

```diff
@@ -142,7 +142,7 @@ func (s *GroupsService) CreateGroup(opt *CreateGroupOptions, options ...OptionFu
 //
 // GitLab API docs:
 // https://docs.gitlab.com/ce/api/groups.html#transfer-project-to-group
-func (s *GroupsService) TransferGroup(gid interface{}, project int, options ...OptionFunc) (*Group, *Response, error) {
+func (s *GroupsService) TransferGroup(gid interface{}, project interface{}, options ...OptionFunc) (*Group, *Response, error) {
```

Comment:
Nice! You should also add something like `project, err := parseID(pid)` to make sure the ID is parsed correctly 👍

Reply:
And then maybe rename the argument from `project` to `pid` to be inline with other ode in this package. Thanks!